source/shared_lib/include/util/gen_uuid.h

```diff
@@ -37,9 +37,11 @@
 //#include "config.h"
 //#endif
 
+#include <stdint.h>
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
+#include <sys/time.h>
```

Comment:
After running clang-format, the build broke with the error that gettimeofday wasn't declared. I added this but of course that doesn't work on Windows.